M2/GNUmakefile.in

```diff
@@ -210,15 +210,7 @@ ifeq (@BUILD_givaro@,yes)
 submodules/fflas_ffpack/Makefile: install-in-givaro
 endif
 
-all-in-Macaulay2:install-licenses
-
-install-licenses:all-in-submodules
-	: installing library and submodule licenses in @pre_licensesdir@
-	if [ -d $(BUILTLIBPATH)/licenses/. ] ; \
-	then rm -rf @pre_licensesdir@ ; \
-	     @INSTALL@ -d @pre_licensesdir@ ; \
-	     @TAR@ cf - -C $(BUILTLIBPATH)/licenses/. . | @TAR@ xf - -C @pre_licensesdir@ ; \
-	fi
```

Comment:
Did the installation of licenses get moved somewhere else?

Reply:
Yes, to libraries/Makefile.in